apps/coordinator/src/components/Wallet/fee-bumping/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { satoshisToBitcoins } from "@caravan/bitcoin";
+
+export const calculateTimeElapsed = (timestamp: number): string => {
+  const now = Date.now();
```

Comment:
the coordinator already has momentjs installed so you could probably use that to do time comparisons a little more easily.